docs/setup/pi4.md

```diff
@@ -0,0 +1,47 @@
+# Connect Create® 3 to Raspberry Pi® 4 and set up ROS 2 Galactic
+
+1. Download [Ubuntu® Server 20.04 64-bit](https://ubuntu.com/download/raspberry-pi) and write onto a microSD card.
+1. In the system-boot partition, edit usercfg.txt and add `dtoverlay=dwc2,dr_mode=peripheral`
+1. In the system-boot partition, edit cmdline.txt to add `modules-load=dwc2,g_ether` after `rootwait`
+1. In the system-boot partition, edit network-config to optionally add information about your Wi-Fi connection, and also add the following under `ethernets`
+
+                usb0:
```

Comment:
nitpick: use ``` rather than all the tabs, especially if the code inside is going to be indented

Reply:
I had quite a bit of trouble with markdown restarting my ordered lists when I used ``` blocks. Feel free to take a pass at it if you feel strongly; otherwise I am going to leave it as it is for now.